collector/cache.go

```diff
@@ -0,0 +1,452 @@
+// +build windows
+
+package collector
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/common/log"
+)
+
+func init() {
+	registerCollector("cache", newCacheCollector, "Cache")
+}
+
+// A CacheCollector is a Prometheus collector for WMI Win32_PerfFormattedData_PerfOS_Cache  metrics
```

Comment:
Old :)

Reply:
Good spot, I've amended the commit to fix this.